src/storage/src/hummock/file_cache/store.rs

```diff
@@ -35,16 +36,81 @@ pub enum FsType {
     Xfs,
 }
 
+pub struct WriteBatch<K, V>
```

Comment:
IMO, a more elegant and rust-style way to express the semantic of batch write will be, we have a `StoreBatchWriter`
```
pub struct StoreBatchWriter<'a, K, V>
where
    K: TieredCacheKey,
    V: TieredCacheValue,
{
    keys: Vec<K>,
    buffer: DioBuffer,
    blocs: Vec<BlockLoc>,

    block_size: usize,

    _phantom: PhantomData<(K, V)>,

    store: &'a mut Store
}
```
and there will be a `start_batch_writer` method in `Store`, which returns a `StoreBatchWriter`.

When `finish` of `StoreBatchWriter` is called, the `StoreBatchWriter` will do the encoding and write to the `Store`. And in this way, the definition of `Store` will not have an extra generic type `V`, and we don't need to add the new `batch` and `insert` method in `Store`.

Reply:
Nice advice! I'll refactor that.